frozen_requirements.txt

```diff
@@ -14,110 +14,107 @@
 git+https://github.com/dcos/dcos-cli.git@f1fd38c9e72e1d521cf8120fe4948a789fb40cbc#egg=dcoscli&subdirectory=cli  # validator-hint: name=dcoscli version=SNAPSHOT
 git+https://github.com/dcos/dcos-cli.git@f1fd38c9e72e1d521cf8120fe4948a789fb40cbc  # validator-hint: name=dcos version=SNAPSHOT
 git+https://github.com/dcos/dcos-test-utils.git@3ebfc18ff9c5a1aa382311474a9192a68c98b0a7  # validator-hint: version=0.1
-git+https://github.com/dcos/dcos-launch.git@dc1e116685fba5105f322b007549b7eb104cf441  # validator-hint: version=0.1.0
+git+https://github.com/dcos/dcos-launch.git@965a877ab641ab39733887188ed8f86d1a8c696c  # validator-hint: version=0.1.0
 adal==1.2.0
 altgraph==0.16.1
 appdirs==1.4.3
 asn1crypto==0.24.0
-aspy.yaml==1.1.1
-astroid==2.1.0
+aspy.yaml==1.2.0
+astroid==2.2.3
 atomicwrites==1.2.1
 attrs==18.2.0
-awscli==1.16.88
+awscli==1.16.117
 azure-common==1.1.16
 azure-mgmt-network==2.0.0
 azure-mgmt-nspkg==3.0.2
 azure-mgmt-resource==2.0.0
 azure-monitor==0.3.1
 azure-nspkg==3.0.2
 azure-storage==0.36.0
-bcrypt==3.1.5
+bcrypt==3.1.6
 black==18.9b0
-boto3==1.9.79
-botocore==1.12.83
-cached-property==1.5.1
+boto3==1.9.76
+botocore==1.12.76
 cachetools==3.0.0
 Cerberus==1.2
 certifi==2018.11.29
 cffi==1.11.5
-cfgv==1.1.0
+cfgv==1.5.0
 chardet==3.0.4
 Click==7.0
-colorama==0.4.1
+colorama==0.3.9
 cryptography==2.4.2
 dcos-shakedown==1.5.0
 docopt==0.6.2
 docutils==0.14
-entrypoints==0.2.3
+entrypoints==0.3
 filelock==3.0.10
-flake8==3.6.0
+flake8==3.7.7
 future==0.17.1
 google-api-python-client==1.7.7
-google-auth==1.6.1
+google-auth==1.6.2
 google-auth-httplib2==0.0.3
 httplib2==0.12.0
-identify==1.1.7
+identify==1.4.0
 idna==2.8
-importlib-resources==1.0.2
 importlib-metadata==0.8
+importlib-resources==1.0.2
 isodate==0.6.0
-isort==4.3.4
-jeepney==0.4
+isort==4.3.10
 jmespath==0.9.3
 jsonschema==2.6.0
 keyring==12.0.2
-keyrings.alt==3.1.1
 lazy-object-proxy==1.3.1
 macholib==1.11
 mccabe==0.6.1
 more-itertools==5.0.0
-msrest==0.6.2
+msrest==0.6.4
 msrestazure==0.4.34
 nodeenv==1.3.3
 oauth2client==3.0.0
 oauthlib==2.1.0
 pager==3.3
 paramiko==2.4.2
+pathlib2==2.3.3
 pefile==2018.8.8
-pkginfo==1.4.2
-pluggy==0.8.0
-pre-commit==1.12.0
+pkginfo==1.2.1
```

Comment:
you meant to downgrade this?

Reply:
yeah it's the result of the `pip freeze` as documented at https://github.com/mesosphere/dcos-commons/blob/master/frozen_requirements.txt#L3-L13